lib/integration/scenarios/Test/Integration/Scenario/API/Byron/Addresses.hs

```diff
@@ -190,8 +195,9 @@ scenario_ADDRESS_LIST_04 fixture = it title $ \ctx -> runResourceT $ do
     r <- request @[ApiAddressWithPath n] ctx (Link.listAddresses @'Byron w) Default Empty
     verify r
         [ expectResponseCode HTTP.status404
-        , expectErrorMessage $ errMsg404NoWallet $ w ^. walletId
         ]
+    decodeErrorInfo r `Lifted.shouldBe`
+        (NoSuchWallet $ ApiErrorNoSuchWallet $ w ^. walletId)
```

Comment:
For cases like this, I would suggest:

```suggestion
        NoSuchWallet (ApiErrorNoSuchWallet $ w ^. walletId)
```

Reply:
(I think `hlint` highlights a series of similar cases.)

Reply:
indeed, done

Reply:
@paweljakubas `hlint` is also failing for these: https://buildkite.com/cardano-foundation/cardano-wallet/builds/4956#018f8702-7a5f-465b-8b49-1b376461a17a